Parallel Compilation Error

When I try to use an embedded Argobots library(`external/argobots`) and compile BOLT in parallel (e.g., `make -j`), it causes the following compile-time error:
```
$ cmake -DLIBOMP_USE_ARGOBOTS=on
...
$ make -j
Performing configure step for 'libabt'
...
bolt/runtime/src/kmp.h:23:17: fatal error: abt.h: No such file or directory
 #include <abt.h>
```

--- a/runtime/src/CMakeLists.txt
+++ b/runtime/src/CMakeLists.txt
@@ -145,6 +145,9 @@ endif()
 libomp_get_ldflags(LIBOMP_CONFIGURED_LDFLAGS)
 
 add_library(omp ${LIBOMP_LIBRARY_KIND} ${LIBOMP_SOURCE_FILES})
+if(${LIBOMP_USE_BUILTIN_ARGOBOTS})
+  add_dependencies(omp libabt)
+endif()
 
 set_target_properties(omp PROPERTIES
   PREFIX "" SUFFIX "" OUTPUT_NAME "${LIBOMP_LIB_FILE}"

